aerogear/digger-android-slave-image
# AGDIGGER-131 Fix symlink to Android debug keystore

### How to verify
1. Build the image, with a tag of your own choice
```
docker build . -t test-jenkins-android-slave
```

2. Change K8s config in Jenkins. Change Android slave image from "docker.io/aerogear/jenkins-android-slave:dev" to "test-jenkins-android-slave"

3. Build an application (you can use https://github.com/aliok/android25sampleapp.git if you don't have one handy). Download the artifact.

4. Do step #3 again.

5. Run this command for both APKs and check of they match (sample command on my machine):
```
/usr/local/Cellar/android-sdk/24.4.1_1/build-tools/25.0.2/apksigner verify --verbose --print-certs ~/Downloads/app-debug14.apk
```

6. Install the first APK on an Android device. Then, without removing it, install the 2nd one. You should be able to install the 2nd w/o problems.
